controllers/users.js

```diff
@@ -174,33 +174,32 @@ const getUsers = async (req, res) => {
     if (transformedQuery?.filterBy === OVERDUE_TASKS) {
       try {
         const tasksData = await getOverdueTasks(days);
+        if (!tasksData.length) {
+          return res.json({
+            message: "No users found",
+            users: [],
```

Comment:
NIT - can we not return the message saying `No overdue task found` ?

Reply:
No, we cannot do that since the API response contains information about returning users.